sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -359,6 +471,50 @@ class RocksDB(
     this
   }
 
+  /**
+   * Initialize in memory values based on the metadata loaded from DFS.
```

Comment:
This is too vague. We should be more concrete. We should be explicit what it does. It is to open local RocksDB and initialize number of key metrics?